src/modules/commentFunctions.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+export const close = () => {
+    const comDiv = document.querySelector('#comment')
+    comDiv.innerHTML = ''
+}
+
+export const commentCounter = (array) => {
+  if(array.length){
+      return array.length
+  } else {
+     return "0"
+  }   
+}
```

Comment:
- [ ] Please know that a counter function should look for specific DOM elements, in this case, the comments, and make the counting based on what is actually displayed on the page, and not based on the array length, kindly fix this.